.github/CODEOWNERS

```diff
@@ -143,30 +143,30 @@ packages/react/src/components/Fabric/ @dzearing
 packages/react/src/components/Facepile/ @markionium @mtennoe
 packages/react/src/components/FolderCover/ @ThomasMichon @bigbadcapers
 packages/react/src/components/FocusTrapZone/ @microsoft/cxe-red @khmakoto
-packages/react/src/components/GroupedList/  @ThomasMichon
-packages/react/src/components/HoverCard/  @Jahnp @Vitalius1
+packages/react/src/components/GroupedList/ @ThomasMichon
+packages/react/src/components/HoverCard/ @Jahnp @Vitalius1
```

Comment:
This one should probably have cxe-red added as well. And maybe check with Vitalie and Peter about whether they still want to be listed at all.

Reply:
Added `@microsoft/cxe-red` here. I'll ask Vitalie and Peter and update CODEOWNERS in a separate PR for that.